### PR TITLE
Some Zayin Buffs and Visual Glow ups.

### PR DESCRIPTION
**Pull request by TaculoTaculo**
## About The Pull Request
Buffs One Sin, Quiet Day, Blubbering Toad, Fairy Festival.
Adds Confession work (again), gamble for a full heal or a medium hit to your sanity (much more likely to get the heal with higher temp)
Toad weh's need a work to activate, sanity regen greatly increased.
Fairy Festival will just pause on work now, regen increased, duration increased.

Adds animations for them, and some effects.

## Why It's Good For The Game
Zayin buffs.
Visual prettyness too.

## Changelog
:cl:
balance: buffs one sin, quiet day, blubbering toad, fairy festival
imageadd: adds new fairy festival, one sin, quiet day, toad and bald 3 sprites
adds status icons for quiet day and toad
adds effect sprites
imagedel: removes old fairy festival and bald 3 sprites
code: changes one sin, toad, quiet day, fairy festival code
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
I would NOT give a +15 stat buff to literally everyone in sight my god

**Comment by TaculoTaculo** [comment]
> I would NOT give a +15 stat buff to literally everyone in sight my god

Change reverted

File: code/modules/mob/living/simple_animal/abnormality/zayin/blubbering_toad.dm [hunk 1]
```diff
@@ -63,20 +65,28 @@
...
-	if(IsContained()) //isn't breached
+	if(IsContained() && (healing_pulse_amount > 0)) //isn't breached and has charges left
+		healing_pulse_amount --
 		HealPulse()
 
 /mob/living/simple_animal/hostile/abnormality/blubbering_toad/proc/HealPulse()
 	for(var/mob/living/L in livinginview(3, get_turf(src)))
 		if(!ishuman(L))
 			continue
 		var/mob/living/carbon/human/H = L
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/zayin/blubbering_toad.dm:88]: Technically not your doing, but:
```suggestion
	for(var/mob/living/carbon/human/H in livinginview(3, get_turf(src)))
```
You can just do that.

File: code/modules/mob/living/simple_animal/abnormality/zayin/fairy_festival.dm [hunk 2]
```diff
@@ -40,16 +40,17 @@
 		if(heal_cooldown <= world.time)
 			P.adjustBruteLoss(-heal_amount*P.getMaxHealth())
 			P.adjustFireLoss(-heal_amount*P.getMaxHealth())
-			P.adjustSanityLoss(-heal_amount*P.getMaxSanity())
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/zayin/fairy_festival.dm:43]: What for?

File: code/modules/mob/living/simple_animal/abnormality/zayin/one_sin.dm [hunk 3]
```diff
@@ -46,40 +57,58 @@
 				A.betrayed = TRUE // So no spam happens
 				for(var/mob/M in GLOB.player_list)
 					if(M.client)
-						M.playsound_local(get_turf(M), 'sound/abnormalities/onesin/confession_start.ogg', 25, 0)
-				return TRUE
-		return FALSE
+						M.playsound_local(get_turf(M), 'sound/abnormalities/onesin/confession_start.ogg', 25, 0) //need to make this non spammable
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/zayin/one_sin.dm:60]: You don't have to keep it for all mobs when it isn't to kill WN, you know?

File: code/modules/mob/living/simple_animal/abnormality/zayin/blubbering_toad.dm [hunk 4]
```diff
@@ -63,20 +65,28 @@
...
+		new /obj/effect/temp_visual/impact_effect/ion(get_turf(L))
```
[the remark this answers is not in the document]
Comment (Coxswain-Navigator) [reply code/modules/mob/living/simple_animal/abnormality/zayin/blubbering_toad.dm:89]: Egor is right, you can remove this effect.

[the remark this answers is not in the document]
Comment (TaculoTaculo) [reply code/modules/mob/living/simple_animal/abnormality/zayin/blubbering_toad.dm:89]: I in fact did not realize, removing, thanks.

File: code/modules/mob/living/simple_animal/abnormality/zayin/fairy_festival.dm (hunk 2 above)
Comment (TaculoTaculo) [reply code/modules/mob/living/simple_animal/abnormality/zayin/fairy_festival.dm:43]: You asked for it, actually

File: code/modules/paperwork/records/info/zayin.dm [hunk 5]
```diff
@@ -70,7 +71,7 @@
 	abno_code = "D-01-145"
 	abno_info = list(
 		"When work with T-01-145 was completed, it started to speak.",
-		"If the agent working with T-01-145 listened to it's entire story, they recieved a buff for the corresponding attribute lasting 5 minutes.")
+		"All agents present on the room while T-01-145 told it's entire story recieved a buff for the corresponding attribute, lasting 5 minutes.")
```
Comment (LanceSmites328) [inline code/modules/paperwork/records/info/zayin.dm:74]: No longer accurate and needs to be reverted.

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/modules/paperwork/records/info/zayin.dm (hunk 5 above)
Comment (TaculoTaculo) [reply code/modules/paperwork/records/info/zayin.dm]: Reverted, instead corrected a spelling mistake

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**